sky/utils/controller_utils.py

```diff
@@ -718,10 +718,19 @@ def maybe_translate_local_file_mounts_and_sync_up(task: 'task_lib.Task',
     if copy_mounts_with_file_in_src:
         # file_mount_remote_tmp_dir will only exist when there are files in
         # the src for copy mounts.
-        storage = task.storage_mounts[file_mount_remote_tmp_dir]
-        store_type = list(storage.stores.keys())[0]
-        store_prefix = store_type.store_prefix()
-        bucket_url = store_prefix + file_bucket_name
+        storage_obj = task.storage_mounts[file_mount_remote_tmp_dir]
+        store_type = list(storage_obj.stores.keys())[0]
+        if store_type is storage_lib.StoreType.AZURE:
+            store_object = cast(storage_lib.AzureBlobStore,
+                                storage_obj.stores[store_type])
+            storage_account_name = store_object.storage_account_name
+            bucket_url = data_utils.AZURE_CONTAINER_URL.format(
+                storage_account_name = storage_account_name,
+                container_name = storage_obj.name
+            ) 
```

Comment:
could we add a util functions for constructing this bucker URL in `storage_lib.AzureBlobStore` class? we could also move the constant to the storage lib

Reply:
We can't move the contant to storage lib since the constant is used by `data_utils` as well. Also, function for just to creating a string seems to be an overshoot.